libr/core/fortune.c

```diff
@@ -94,16 +94,27 @@ R_API void r_core_fortune_list(RCore *core) {
 }
 
 static char *getrandomline(RCore *core) {
-	const char *types = (char *)r_config_get (core->config, "cfg.fortunes.type");
+	RList *types = r_str_split_duplist (
+		(char *)r_config_get (core->config, "cfg.fortunes.type"), ",", false);
+	if (!r_list_length (types)) {
+		r_list_free (types);
+		return NULL;
+	}
+	char *type = r_str_new ((char *)r_list_get_n (types, r_num_rand (r_list_length (types))));
```

Comment:
assuming that cant return null

Reply:
what can't return NULL?

Reply:
List.getN can return null if passing the wrong index.. which shouldnt happen.. but if the array is empty the index is 0 or -1 and that will fail i think. So better null check imho

Reply:
fix this and ok to merge